test/executor.test.js

```diff
@@ -306,15 +307,25 @@ describe('executor', function() {
           name: 'Timestamping'
         });
 
-        var options = {
-          appRootDir: appdir.PATH,
-          mixinDirs: ['./mixins']
+        options = {
+          appRootDir: appdir.PATH
         };
+      });
 
+      it('defines mixins from instructions - using `mixinDirs`', function() {
+        options.mixinDirs = ['./mixins'];
```

Comment:
Hmm, once `mixinSources` are correctly implemented, this test always passes, because `./mixins` is the default value of `mixinSources`. Isn't it so? I think you need to change this test to use `./custom-mixins` instead.